src/key.rs

```diff
@@ -259,7 +259,7 @@ impl KeyUpdater {
     /// let mut key_update = KeyUpdater::new(&key.key);
     /// key_update.with_description(&description).execute(&client).await;
     ///
-    /// assert_eq!(key_update.description, Some(description));
+    /// # assert_eq!(key_update.description, Some(description));
```

Comment:
Why do you want to hide this assert? It shows the update worked well I think?

And it’s pretty common all over the documentation, look at the [push method from the stdlib](https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.push)